lace/emulator/p1d_archive.py

```diff
@@ -11,14 +11,24 @@
 class archiveP1D(object):
     """Book-keeping of flux P1D measured in a suite of simulations."""
 
-    def __init__(self,basedir="/lace/emulator/sim_suites/Australia20/",
-                p1d_label=None,skewers_label=None,
-                drop_tau_rescalings=True,drop_temp_rescalings=True,
-                keep_every_other_rescaling=False,nearest_tau=False,
-                max_archive_size=None,undersample_z=1,verbose=False,
-                no_skewers=False,pick_sim_number=None,drop_sim_number=None,
-                drop_snap_number=None,z_max=5.,nsamples=None,undersample_cube=1,
-                kp_Mpc=None,drop_redshift=None,pick_redshift=None):
+    def __init__(
+		self,
+		basedir="/lace/emulator/sim_suites/Australia20/",
+                p1d_label=None,
+		skewers_label=None,
+                drop_tau_rescalings=True,
+		drop_temp_rescalings=True,
+                verbose=False,
+                no_skewers=False,
+		pick_sim_number=None,
+		drop_sim_number=None,
+                drop_snap_number=None,
+		z_max=4.5,
```

Comment:
Here you have changed the z_max default, we should be careful with this as well.